pom.xml

```diff
@@ -120,7 +120,7 @@
         <dependency>
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-databind</artifactId>
-            <version>2.9.10.1</version>
+            <version>2.9.10.2</version>
```

Comment:
What's this change for?

Reply:
Whitesource marked this dep as vulnerable